_data/heroku-landing-page-tabs-code.yml

```diff
@@ -0,0 +1,111 @@
+- id: fargate
+  title: Fargate
+  content: |
+    <figure><figcaption>ecs-fargate-service/main.tf</figcaption>
+      <pre>
+      <code class="language-js">
+        module "fargate_service" {
+          source = "git::git@github.com:gruntwork-io/terraform-aws-ecs.git//modules/ecs-service?ref=v1.0.8"
```

Comment:
NIT: we should probably be using the `ecs-service` from `terraform-aws-service-catalog`.

Reply:
I recently used this in my own experiments, so here is a simple snippet you can use that highlights the most important features:

```hcl
module "fargate_service" {
  source = "git::git@github.com:gruntwork-io/terraform-aws-service-catalog.git//modules/services/ecs-service?ref=v0.51.0"  

  service_name          = "my-heroku-app"
  launch_type           = "FARGATE"
  ecs_cluster_arn       = module.fargate_cluster.arn 
  
  task_cpu                = 256
  task_memory             = 512
  desired_number_of_tasks = 3

  container_definitions = [
    {
      name         = "my-heroku-app"
      image        = "https://123456789012.dkr.ecr.us-east-1.amazonaws.com/my-heroku-app:v1"
      essential    = true
      portMappings = [
        {
          "hostPort"      = 3000
          "containerPort" = 3000
          "protocol"      = "tcp"
        }
      ]      
    }
  ]

  domain_name = "example.your-domain.com"
}

module "fargate_cluster" {
  source = "git::git@github.com:gruntwork-io/terraform-aws-service-catalog.git//modules/services/ecs-fargate-cluster?ref=v0.51.0"  

  cluster_name = "my-fargate-cluster"
}
```